packages/vats/test/test-gov-starting.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+import test from 'ava';
+import { createRequire } from 'module';
+import fsp from 'fs/promises';
+
+import { permit } from '../src/core/gov-starting.js';
+
+const myRequire = createRequire(import.meta.url);
+
+/**
+ * If $SCRIPT is set, the test(s) below will write the rendered script to a file
+ * by that name. Likeiwse $PERMIT.
+ */
+test.before(async t => {
+  t.context = {
+    readFile: fsp.readFile,
+    env: process.env,
+    writeFile: fsp.writeFile,
+  };
+});
+
+test('check / save gimix permit', async t => {
+  t.notThrows(() => JSON.parse(permit));
+
+  // IDEA: check permit against a pattern
+
+  // @ts-expect-error TODO: types of t.context
+  const { env } = t.context;
+  if (env.PERMIT) {
+    // @ts-expect-error TODO: types of t.context
+    const { writeFile } = t.context;
+    const pretty = JSON.stringify(JSON.parse(permit), null, 2);
+    t.log('write permit to', env.PERMIT);
+    await writeFile(env.PERMIT, pretty);
+  }
+});
+
+const redactImportDecls = txt =>
```

Comment:
comment here that this is a work-around for lack of typedef of core-eval environment (which has `E`, `harden`, etc) https://github.com/Agoric/agoric-sdk/issues/8643